src/pages/home/category-item/category-item.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { useStyles } from './category-item.style';
+
+const Categories = ({ categoryName, categoryImage, categoryCode }) => {
+  const props = {
```

Comment:
Why do you need this variable? Can you just write `useStyles({ image: categoryImage })` and get rid of `props`

Reply:
Fixed